SP23.P03.Web/Controllers/TripsController.cs

```diff
@@ -0,0 +1,177 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SP23.P03.Web.Data;
+using SP23.P03.Web.Extensions;
+using SP23.P03.Web.Features.Authorization;
+using SP23.P03.Web.Features.Trains;
+using SP23.P03.Web.Features.TrainStations;
+using SP23.P03.Web.Features.Trips;
```

Comment:
```suggestion
﻿using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.Mvc;
using Microsoft.EntityFrameworkCore;
using SP23.P03.Web.Data;
using SP23.P03.Web.Features.Authorization;
using SP23.P03.Web.Features.Trains;
using SP23.P03.Web.Features.TrainStations;
using SP23.P03.Web.Features.Trips;
```
Unused imports according to VS